### PR TITLE
NF: converting Long to DeckId

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Those one were found with searcing for "eckId: Long"

### CODE REVIEW

**Review by lukstbit — APPROVED** [review]
Ok, but to be honest I'm not very fond of `did: did`.

**Comment by dae** [comment]
Me either - lowercase type names don't seem to be idiomatic in Kotlin, and "did: DeckId" would make it much more readable for people not familiar with Anki's abbreviations.

**Comment by Arthur-Milchior** [comment]
Modified. In this case, I note that Python originally use `did` but it's renamed for consistency with Kotlin style. 
I love having proper type. I don't like having two types with no semantic difference

File: AnkiDroid/src/main/java/com/ichi2/libanki/PythonTypes.kt [hunk 1]
```diff
@@ -26,7 +26,10 @@ import java.util.*
 internal typealias Dict<K, V> = HashMap<K, V>
 internal typealias ImmutableList<T> = List<T>
 internal typealias str = String
-internal typealias did = Long
+/**
+ * Equivalent to `did` in Python.
+ */
+internal typealias DeckId = Long
```
Comment (dae) [inline AnkiDroid/src/main/java/com/ichi2/libanki/PythonTypes.kt:32]: I don't think `did` has even been used as a type name. It was (and still is) used as an argument name, but when typing was added, it started off as an int, then went to DeckID and then DeckId. So they're not technically equivalent. If you wanted to keep the comment, perhaps "DeckId arguments are sometimes abbreviated as `did`"?

**Review by dae — COMMENTED** [review]
Apart from my comment about the comment, all looks good.

**Review by david-allison — APPROVED** [review]

**Comment by Arthur-Milchior** [comment]
Comment corrected. Thanks.
No idea why I thought `did` was a Python type. I should have spent enough time there to recall how it worked.

**Pull request merged**